test/runtests.jl

```diff
@@ -68,7 +105,7 @@ const DATADIR = joinpath(@__DIR__, "data")
                         [TimestampedAnnotationList(4.0, nothing, String[""]), TimestampedAnnotationList(2.5, 2.5, ["type A"])],
                         [TimestampedAnnotationList(5.0, nothing, String[""])]]
             @test all(signal.records .== expected)
-            @test AnnotationsSignal(signal.records).samples_per_record == 16
+            @test AnnotationsSignal(signal.records).samples_per_record == 14
```

Comment:
wait, why does this change?

Reply:
this confused me too, but this is actually a number associated to how many bytes we need to serialize, and that number went down with this PR due to more compact printing (e.g. consistently dropping trailing 0s)